tests/ci/cdk/cdk/accp_github_ci_stack.py

```diff
@@ -27,6 +27,8 @@ def __init__(self,
             fetch_submodules=True,
             webhook_filters=[
                 codebuild.FilterGroup.in_event_of(
+                    codebuild.EventAction.PUSH,
+                    codebuild.EventAction.PULL_REQUEST_MERGED,
```

Comment:
Are both of these necessary, or would `PUSH` be enough to suffice?

Reply:
i'm not 100% sure, but the fact that they bothered to break them out into separate fields indicates to me that GitHub differentiates between a "push" action and a "merge form PR" action. in either case, i feel there's no potential harm in including both.